app/src/main/java/bluedot/com/au/bluedotbrazeintegrationapp/BluedotGeoTriggerReceiver.kt

```diff
@@ -2,47 +2,46 @@ package bluedot.com.au.bluedotbrazeintegrationapp
 
 import android.content.Context
 import au.com.bluedot.point.net.engine.GeoTriggeringEventReceiver
-import au.com.bluedot.point.net.engine.ZoneInfo
 import au.com.bluedot.point.net.engine.event.GeoTriggerEvent
 import com.braze.Braze
 import com.braze.models.outgoing.BrazeProperties
-import java.util.*
+import java.util.Date
 
 class BluedotGeoTriggerReceiver: GeoTriggeringEventReceiver() {
     private val customEventEntry = "bluedot_entry"
     private val customEventExit = "bluedot_exit"
 
-    override fun onZoneEntryEvent(geoTriggerEvent: GeoTriggerEvent, context: Context) {
-        println("Zone ${geoTriggerEvent.zoneInfo.name}, fence ${geoTriggerEvent.entryEvent()?.fenceName} entered at: ${Date()}")
+    override fun onZoneEntryEvent(entryEvent: GeoTriggerEvent, context: Context) {
+        println("Zone ${entryEvent.zoneInfo.name}, fence ${entryEvent.entryEvent()?.fenceName} entered at: ${Date()}")
 
         val eventProperties = BrazeProperties()
-        eventProperties.addProperty("zone_id", geoTriggerEvent.zoneInfo.id)
-        eventProperties.addProperty("zone_name", geoTriggerEvent.zoneInfo.name)
+        eventProperties.addProperty("zone_id", entryEvent.zoneInfo.id.toString())
+        eventProperties.addProperty("zone_name", entryEvent.zoneInfo.name)
         eventProperties.addProperty("latitude",
-            geoTriggerEvent.entryEvent()?.locations?.get(0)?.latitude
+            entryEvent.entryEvent()?.locations?.get(0)?.latitude
         )
-        eventProperties.addProperty("longitude", geoTriggerEvent.entryEvent()?.locations?.get(0)?.longitude)
-        eventProperties.addProperty("fence_id", geoTriggerEvent.entryEvent()?.fenceId)
-        eventProperties.addProperty("fence_name", geoTriggerEvent.entryEvent()?.fenceName)
+        eventProperties.addProperty("longitude", entryEvent.entryEvent()?.locations?.get(0)?.longitude)
```

Comment:
[I] Do we need to change geoTriggerEvent -> entryEvent?
Using double of `entryEvent.entryEvent` that actually have different types looks a bit confusing.

Reply:
I renamed the `geotriggerEvent` to match the syntax as returned in the docs:
https://android-docs.bluedot.io/-bluedot%20-s-d-k%20-docs/au.com.bluedot.point.net.engine/-geo-triggering-event-receiver/index.html
Android studio also reports warnings if same name is not used as mentioned in the Docs.
I see it's creating a confusion `entryEvent.entryEvent` but it would better to match syntax with the docs and have no warnings in the code.
Confusion can be cleared after referring the docs for `entryEvent`: https://android-docs.bluedot.io/-bluedot%20-s-d-k%20-docs/au.com.bluedot.point.net.engine.event/-geo-trigger-event/index.html#-1077835187%2FFunctions%2F-63584292